### PR TITLE
Backport GeckoPackages/DiffOutputBuilder

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by SpacePossum** [comment]
merged

File: LICENSE_DIFF [hunk 1]
```diff
@@ -0,0 +1,33 @@
+sebastian/diff
```
Comment (keradus) [inline LICENSE_DIFF:1]: ?
cc @SpacePossum

File: README.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
 # PHP-CS-Fixer/diff
 
-Fork of sebastian/diff
+This is version is for PHP CS Fixer only! Do not use it!
```
Comment (keradus) [inline README.md:3]: `is` x 2
cc @SpacePossum

File: LICENSE_DIFF (hunk 1 above)
Comment (SpacePossum) [reply LICENSE_DIFF:1]: thanks, corrected!

File: README.md (hunk 2 above)
Comment (SpacePossum) [reply README.md:3]: thanks, corrected!

**Pull request closed without merging**